src/checkers/inference/InferenceOptions.java

```diff
@@ -69,6 +71,9 @@
     @Option("[InferenceSolver] the fully-qualified name of the solver to use on constraints; overrides --typesystem.")
     public static String solver;
 
+    @Option("The fully-qualified name of the classpath for target program; overrides --targetclasspath.")
```

Comment:
The other options have a [XXX] term as first part of the string. What's the function of that? Should this string also have some [TargetClassPath] or something?